src/api/views.py

```diff
@@ -72,8 +73,8 @@ def check_domain_available(domain):
         else:
             # domain search string doesn't end with .gov, add it on here
             return Domain.available(domain + ".gov")
-    except Exception:
-        return False
+    except Exception as err:
```

Comment:
I think since you are just raising the error here you can just remove the try/except block entirely